Site/_config.yml

```diff
@@ -0,0 +1,58 @@
+url: "https://m4rs-mt.github.io"
+baseurl: "/ILGPU"
```

Comment:
Should be updated in order to use the domain: http://ilgpu.net/

Reply:
I think we should point to https://www.ilgpu.net/

Reply:
Yes, but we will need the to configure the DNS as well (CNAME file)